libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/components/list/ListItemContent.kt

```diff
@@ -114,12 +121,28 @@ sealed interface ListItemContent {
                 onClick = onClick,
                 enabled = enabled
             )
-            is Icon -> IconComponent(
-                modifier = Modifier.size(maxCompactSize),
-                painter = iconSource.getPainter(),
-                contentDescription = iconSource.contentDescription
-            )
+            is Icon -> {
+                val iconComponent = @Composable {
+                    IconComponent(
+                        modifier = Modifier.size(maxCompactSize),
+                        painter = iconSource.getPainter(),
+                        contentDescription = iconSource.contentDescription
+                    )
+                }
+                if (showBadge) {
```

Comment:
Thanks for migrating the support of `showBadge`. We can keep it, but I think it's not used anymore. The RedIndicatorAtom is now displayed at the end of the item (design decision change...)